airflow/providers/google/cloud/hooks/datastore.py

```diff
@@ -279,11 +270,10 @@ def export_to_storage_bucket(
         :param labels: Client-assigned labels.
         :param project_id: Google Cloud project ID against which to make the request.
         :return: a resource operation instance.
-        :rtype: dict
         """
-        admin_conn = self.get_conn()  # type: Any
+        admin_conn = self.get_conn()
 
-        output_uri_prefix = "gs://" + "/".join(filter(None, [bucket, namespace]))  # type: str
+        output_uri_prefix: str = "gs://" + "/".join(filter(None, [bucket, namespace]))
```

Comment:
```suggestion
        output_uri_prefix = f"gs://{'/'.join(filter(None, [bucket, namespace]))}"
```

Reply:
Yep better. Did that for both of them, also renamed the local variable `output_uri_prefix` to `output_url_prefix` for consistency. It is used as an `outputUrlPrefix`...
```python
        body = {
            "outputUrlPrefix": output_url_prefix,
            "entityFilter": entity_filter,
            "labels": labels,
        }
```

cf: https://github.com/apache/airflow/commit/409818b7450d57fb79a29a71c6be98a4b3adb6b8